### PR TITLE
Move version non-specific parts of message body into semantics

### PR DESCRIPTION
**Pull request by mnot**
For #159.

### CODE REVIEW

**Comment by mnot** [comment]
Roughed in; PTAL.

File: draft-ietf-httpbis-messaging-latest.xml [hunk 1]
```diff
@@ -939,39 +939,29 @@ https://www.example.org
...
 </sourcecode>
 <t>
-   The rules for when a message body is allowed in a message differ for
-   requests and responses.
+   The rules for determining when a message body is present in a HTTP/1.1
+   message differ for requests and responses.
 </t>
 <t>
-   The presence of a message body in a request is signaled by a
+   The presence of a message body in a HTTP/1.1 request is signaled by a
```
Comment (reschke) [inline draft-ietf-httpbis-messaging-latest.xml]: Not sure we need to say HTTP/1.1 everywhere - this is the topic of this spec, right?

Comment (mnot) [reply draft-ietf-httpbis-messaging-latest.xml]: I took that one out, but left the first one to remind readers that this is 1.1-specific.

**Comment by mnot** [comment]
I don't any requirements were lost, they're just incorporated by reference, since they apply to more than one version of HTTP.

File: draft-ietf-httpbis-messaging-latest.xml [hunk 2]
```diff
@@ -939,17 +939,17 @@ https://www.example.org
...
+   message body is identical to the payload body unless a transfer coding has
+   been applied, as described in <xref target="header.transfer-encoding"/>.
 </t>
 <sourcecode type="abnf7230"><iref primary="true" item="Grammar" subitem="message-body"/>
   <x:ref>message-body</x:ref> = *OCTET
 </sourcecode>
 <t>
-   The rules for when a message body is allowed in a message differ for
-   requests and responses.
+   The rules for determining when a message body is present in a HTTP/1.1
```
Comment (MikeBishop) [inline draft-ietf-httpbis-messaging-latest.xml]:
```suggestion
   The rules for determining when a message body is present in an HTTP/1.1
```

**Pull request merged**